.github/workflows/docker.yml

```diff
@@ -58,9 +56,8 @@ jobs:
           context: .
           platforms: ${{ env.PLATFORMS }}
           push: true
-          tags: |
-            ghcr.io/${{ env.GHCR_NAMESPACE }}/sliding-sync:latest
```

Comment:
We seem to be losing the `:latest` tag..?

Reply:
So the latest tag is no longer specified by docker/build-push-action@v4, but a list of tags (in this case I think only one) is created by docker/metadata-action@v5

See https://github.com/matrix-org/sliding-sync/blob/8bcd28b6c2c33b5b9a42a36cc0ffef3126a241fa/.github/workflows/docker.yml#L46

It'll create `main` on main branch, latest on release, and then always a sha specific tag